backend/codesirius/problems/serializers/sample_test.py

```
@@ -0,0 +1,139 @@
+            return validated_data, None
+        for field, message in serializer.errors.items():
+            errors.append(
+                {
+                    "field": field,
+                    "message": message[0],
+                }
+            )
+        return None, errors
+
+    @staticmethod
+    def _validate_create(attrs) -> tuple[list[ValidData] | None, list[Error] | None]:
```

Ruff (0.8.2)

Missing type annotation for function argument `attrs`

(ANN001)